spec/support/fixtures/vcr_cassettes/portal/generate_link_audit_logs.yml

```diff
@@ -5,7 +5,7 @@ http_interactions:
       uri: https://api.workos.com/portal/generate_link
       body:
         encoding: UTF-8
-        string: '{"intent":"audit_logs","organization":"org_01EHQMYV6MBK39QC5PZXHY59C3","return_url":null}'
+        string: '{"intent":"audit_logs","organization":"org_01EHQMYV6MBK39QC5PZXHY59C3","return_url":null,"success_url":null}'
```

Comment:
Couldn't run the tests locally on my machine so I manually added the  `success_url` here and below in the other two files